samwise.js

```diff
@@ -1,33 +1,31 @@
 /**
```

Comment:
SCRIPT-The CSS rules here seem to be redundant. Can you please review and remove unnecessary ones?